rpc/src/lib.rs

```diff
@@ -67,15 +67,15 @@ impl Builder {
     /// New builder pre-populated with RPC info methods
     pub fn new(rpc_config: RpcConfig) -> Self {
         // TODO: this is a mess because of the configuration file mapping from files to options. See #446
```

Comment:
just a reminder to clean up todos if the PR is merged

Reply:
Thank you.